index.html

```diff
@@ -4,30 +4,43 @@
 		<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
 		<link href="style.css" rel="stylesheet" type="text/CSS">
 	</head>
-	<body>
+	<body id="top">
+		<div class="contact">
```

Comment:
This all looks good, just unsure why this is at the top when it is bottom of the document flow?